test/celeritas/optical/OpticalCollector.test.cc

```diff
@@ -591,8 +587,8 @@ TEST_F(LArSphereOffloadTest, host_generate)
     if (CELERITAS_REAL_TYPE == CELERITAS_REAL_TYPE_DOUBLE)
     {
         EXPECT_SOFT_NEAR(
-            462263.0, static_cast<double>(result.accum.steps), 1e-4);
-        EXPECT_EQ(37, result.accum.step_iters);
+            4593666, static_cast<double>(result.accum.steps), 1e-4);
```

Comment:
I'm not sure why this value (sum of the number of active tracks over all step iterations) changed by an order of magnitude, but given the number of step iterations and track slots the new value seems more reasonable.